src/components/ListPage/ListPage.ts

```diff
@@ -170,6 +171,10 @@ export namespace ListPage {
       return (this.getSingleQueryParam('direction') || 'asc') === 'asc';
     }
 
+    currentSortFieldId(): string | undefined {
```

Comment:
@xeviknal I think we could do something better here but I restrained to do it since you're doing some UI refactoring, that would more likely fall into your domain.
What I thought was:, in `ListPage`, having `currentSortField(): SortField` instead of `currentSortFieldId(): string | undefined`
but that's currently impossible due to `SortField` being different classes depending on the context (one for apps, one for workloads etc.)
I believe with the refactoring we could have a common interface `SortField<T>` for all of them. Do you agree?

Reply:
@jotak I do agree. I am playing a bit with `SortField<T>` parameterized type. What I have doubts if TS will allow us to return `SortField` when we have `SortField<T>` as a type.

Reply:
No it's fine with generics then the signature would be `currentSortField(): SortField<T>` which is really fine (maybe the whole `ListPage` would have to have parameter `<T>` ... I think it can work, let me know / chat if there's some troubles with that)